### ISSUE TITLE
stop playing sound

### ISSUE THREAD

**Issue by arist0v** [TaylorSMarks/playsound#139]
should have a way to stop the sound from playing (ex in a tkinter windows if i close the windows i didn't want the sound to play anymore)

**Comment by AlkisPis** [comment]
Certainly. A program in **Python** in which you have to press Ctrl+Break (not even Ctrl+C) to terminate is unacceptable. (In other programmin languages, one might not have other  choice.)
